app/models/game.rb

```diff
@@ -6,4 +6,5 @@ class Game < ApplicationRecord
   belongs_to :black_player, class_name: 'User'
 
   enum game_status: { in_progress: 0, checkmate: 1, stalemate: 2 }
+  scope :available, -> { where(available: true) }
```

Comment:
Where/when does `available` get set?

Is it worth adding another column to the database if you can get the info from `black_player_id` and `white_player_id`